simtools/applications/generate_default_metadata.py

```diff
@@ -8,7 +8,7 @@
     ----------------------
     schema (str, optional)
         Schema file describing the input data
-        (default: simtools/schemas/metadata.schema.yml)
+        (default: simtools/schemas/metadata.metaschema.yml)
```

Comment:
By doing a `grep -r "metadata.schema"` in the main directory I still see some references to `metadata.schema` in `tests/unit_tests/data_model/test_metadata_collector.py` and `tests/unit_tests/data_model/test_metadata_model.py`. I believe we want to change those too.

Reply:
Which one do you mean? Those like `array_coordinates.schema.yml` needs to stay, as they are actual schemas (and not schemas describing schemas).

Reply:
No, I meant these lines which you can get if you run `grep -r "metadata.schema"`. These lines still refer to schema of the metadata, and we now know that they are metaschema, so it is appropriate to change the names there too:

- https://github.com/gammasim/simtools/blob/8bfda9f58c77f76f52a195ca8d6a6b9fa4fa73ef/tests/unit_tests/data_model/test_metadata_collector.py#L62
- https://github.com/gammasim/simtools/blob/8bfda9f58c77f76f52a195ca8d6a6b9fa4fa73ef/tests/unit_tests/data_model/test_metadata_collector.py#L66
- https://github.com/gammasim/simtools/blob/8bfda9f58c77f76f52a195ca8d6a6b9fa4fa73ef/tests/unit_tests/data_model/test_metadata_model.py#L22-L24

Reply:
I think you have a different or outdated branch on disk, I cannot find these issues. Can you check again?